core/services/kraken/manifest/manifest.py

```diff
@@ -0,0 +1,134 @@
+import dataclasses
+import datetime
+from typing import List, Optional
+
+# Extra
+import aiohttp
+from dataclass_wizard import fromdict
+
+# Models
+from manifest.models import ExtensionVersion, RepositoryEntry
+
+# Current Extension Manifest host
+REPO_URL = "https://bluerobotics.github.io/BlueOS-Extensions-Repository/manifest.json"
+
+
+@dataclasses.dataclass
+class ManifestContent:
+    """
+    Represents the manifest content with cache control.
+
+    Attributes:
+        extensions (List[RepositoryEntry]): List of extensions in the manifest.
+        issued_at (Optional[int]): Timestamp when the manifest was issued.
+        expires_in (Optional[int]): Time in seconds until the manifest expires.
+    """
+
+    extensions: List[RepositoryEntry]
+    expires_in: int = 3600
+    issued_at: int = int(datetime.datetime.now(datetime.timezone.utc).timestamp())
+
+    @property
+    def is_expired(self) -> bool:
+        """
+        Checks if the manifest is expired.
+
+        Returns:
+            bool: True if the manifest is expired, False otherwise.
+        """
+
+        return (int(datetime.datetime.now(datetime.timezone.utc).timestamp()) - self.issued_at) > self.expires_in
+
```

Comment:
Consider unifying the timestamp into something like:

```Python
@classmethod
def _current_timestamp(cls) -> int:
  return int(datetime.datetime.now(datetime.timezone.utc).timestamp())
```

Reply:
Added!